docs/CHANGELOG.asciidoc

```diff
@@ -8,53 +8,641 @@
+
+* `alert_update` is now `rule_update`
+
+* `alert_update_api_key` is now `rule_update_api_key`
+
+* `alert_enable` is now `rule_enable`
+
+* `alert_disable` is now `rule_disable`
+
+* `alert_delete` is now `rule_delete`
+
+* `alert_find` is now `rule_find`
+
+* `alert_mute` is now `rule_mute`
+
+* `alert_unmute` is now `rule_unmute`
+
+* `alert_instance_mute` is now `rule_alert_mute`
+
+* `alert_instance_unmute` is now `rule_alert_unmute`
+
+The terminology in the audit messages now references `rules` instead of `alerts`. For example, `User is creating alert [id=${ruleId}]` is now `User is creating rule [id=${ruleId}]`.
+
+For more information, refer to {kibana-pull}102263[#102263].
+
+*Impact* +
+When you upgrade to 7.14.0, {kib} automatically uses the new terminology. No action is needed.
+====
+      
+[discrete]
+[[breaking-99078]]
+.Refactors alerts authorization client
+[%collapsible]
+====
+*Details* +
+The alerts authorization client has been refactored to be more generic. {kib} uses the existing alerting feature privilege model to provide `all` and `read` privileges to `rules` and `alerts`. 
+
+There are no changes to the rules role-based access control (RBAC).
+
+The following changes were made:
+
+* Updated alerting feature privilege builder to include `entity` in the feature privilege string. Allowed entities are `rule` and `alert`.
+
+* Added ability to specify `exemptConsumers` in authorization client to handle the alerting use case where `alerts` was a special consumer ID with special handling.
+
+* Updated terminology in the `AlertsAuthorization` class to reflect the changed rule terminology.
+
+* Updated authorization filter builder to return EsQuery or KQL.
+
+* Updated authorization filter to allow specified field names.
+
+* `getAlertingAuthorizationWithRequest` is now exposed through alerting plugin start, which allows other plugins to use it.
+
+* `AlertsAuthorization` is now `AlertingAuthorization`.
+
+The formatting for the following recorded audit events was changed:
+
+* Audit event types now use `alerting_ prefix` instead of `alerts_ prefix`:
+
+** `alerts_authorization_failure` is now `alerting_authorization_failure`
+
+** `alerts_unscoped_authorization_failure` is now `alerting_unscoped_authorization_failure`
+
+** `alerts_authorization_success` is now `alerting_authorization_success`
+
+* Audit events have been updated to reflect the new alerting terminology:
+
+** `alerts` are now `rules`. For example, the audit event `user Authorized to create a ".index-threshold" alert by "app"` is now `user Authorized to create a ".index-threshold" rule by "app"`.
+
+** `alert instances` are now `alert`. For example, audit events for `muteInstance` and `unmuteInstance` operations now reference `muteAlert` and `unmuteAlert`.
+
+For more information, refer to {kibana-pull}99078[#99078].
+
+*Impact* +
+When you upgrade to 7.14.0, {kib} automatically reflects the changes. No action is needed.
+====    
```

Comment:
@ymao1 does this "refactors alerts authorization client" section look good to you for the 7.14 release notes? It's copied from your PR: https://github.com/elastic/kibana/pull/99078.

Reply:
I think only the section regarding audit events needs to be in the release notes. The breaking change in this case is the fact that alerting authorization audit events now say rule instead of alerts. Everything else in the PR description can be left out.

Reply:
Thanks! So I think the below change will suffice 🙏 

```suggestion
[discrete]
[[breaking-99078]]
.Refactors alerts authorization client
[%collapsible]
====
*Details* +
The alerts authorization client has been refactored to be more generic. {kib} uses the existing alerting feature privilege model to provide `all` and `read` privileges to `rules` and `alerts`. 

For more information, refer to {kibana-pull}99078[#99078].

*Impact* +
When you upgrade to 7.14.0, {kib} automatically reflects the changes. No action is needed.
====    
```

Reply:
What about this to highlight specific audit language changes:

```suggestion
[discrete]
[[breaking-99078]]
.Refactors alerts authorization client
[%collapsible]
====
*Details* +
The terminology has changed for audit events logged by the alerting authorization client client when CRUD operations are performed on rules.

** `alerts` are now `rules`. For example, the audit event `user Authorized to create a ".index-threshold" alert by "app"` is now `user Authorized to create a ".index-threshold" rule by "app"`.

** `alert instances` are now `alert`. For example, audit events for `muteInstance` and `unmuteInstance` operations now reference `muteAlert` and `unmuteAlert`.

For more information, refer to {kibana-pull}99078[#99078].

*Impact* +
When you upgrade to 7.14.0, {kib} automatically reflects the changes. No action is needed.
====    
```